packages/fms-base/conf/Dockerfile.j2

```diff
@@ -0,0 +1,37 @@
+{% if arch %}
+FROM base-builder:{{ codename }}-{{ arch }}-latest
+{% else %}
+FROM base-builder:{{ codename }}-latest
+{% endif %}
+LABEL maintainer="{{ vendor}} <{{ vendor_email }}>"
+
+RUN apt-get -qq update && apt-get -qq -y install perl postgresql
+
+WORKDIR /home/builder
+USER builder
+
+RUN git clone --recursive https://github.com/mysociety/fixmystreet.git
+WORKDIR /home/builder/fixmystreet
+RUN git checkout {{ commit }}
+RUN echo "SHORT_COMMIT=$(git rev-parse --short HEAD)" >> /home/builder/.env
+RUN echo "TIMESTAMP=$(date +%Y%m%d%H%M)" >> /home/builder/.env
+
+USER root
+RUN cp conf/packages.docker /opt/
+RUN xargs -a "/opt/packages.docker" apt-get -qq -y install >/dev/null
+
+USER builder
+RUN ./script/bootstrap
+RUN echo "SHORT_COMMIT=$(git rev-parse --short HEAD)" >> /home/builder/.env
+RUN fpm -s dir -t deb \
+      --vendor "mySociety" \
+      --maintainer "mySociety <sysadmin@mysociety.org>" \
+      --version "{{ version }}+git{{ timestamp }}.${SHORT_COMMIT}" \
+      --iteration "ms1" \
+      --architecture "native" \
+      --url "https://github.com/mysociety/fixmystreet" \
+      --description "mysociety's package containing build artefacts to speed up the process of deploying new FixMyStreet servers." \
+      -n "fms-base" \
+      ./local/=/opt/fixmystreet/perl
```

Comment:
I wonder, do we need the `perl/` subdirectory?

That gives us paths like `/opt/fixmystreet/perl/lib/perl5/` which feels a bit redundant.

Also, do we need the `cache/` directory?